src/data/media-player.ts

```diff
@@ -20,9 +36,38 @@ export const CONTRAST_RATIO = 4.5;
 
 export type MediaPlayerBrowseAction = "pick" | "play";
 
+export const BROWSER_SOURCE = "browser";
+
+export type MediaTypeBrowserSetting = { icon: string };
+
+export const MediaTypeBrowserSettings: {
+  [type: string]: MediaTypeBrowserSetting;
+} = {
+  file: { icon: mdiFile },
+  library: { icon: mdiFolder },
+  playlist: { icon: mdiPlaylistMusic },
+  track: { icon: mdiFileMusic },
+  album: { icon: mdiAlbum },
+  music: { icon: mdiMusic },
+  tvshow: { icon: mdiTelevisionClassic },
+  movie: { icon: mdiMovie },
+  video: { icon: mdiVideo },
+  episode: { icon: mdiFolder },
+  channel: { icon: mdiFolder },
+  image: { icon: mdiImage },
+  url: { icon: mdiWeb },
+  game: { icon: mdiGamepadVariant },
+  app: { icon: mdiFolder },
+  artist: { icon: mdiAccountMusic },
+  contributing_artist: { icon: mdiFolder },
+  podcast: { icon: mdiPodcast },
+  season: { icon: mdiFolder },
+  genre: { icon: mdiFolder },
+  composer: { icon: mdiFolder },
+};
```

Comment:
Unused right now? Let's do this in a new PR where we will use it.

Reply:
Yea I was wondering if we were going to wait on that or not.